contrib/flutter_rust_bridge_codegen.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "1.37.1",
```

Comment:
Btw is this auto-generated by script or manually written? If manually, we may need justfile's `release` recipe to bump all these versions

Reply:
I made it so that `just release` will automatically update this file, and Scoop users can use the auto-update definitions so they don't have to re-download this manifest.

Reply:
In the future, this file should live in the main Scoop bucket repository and managed automatically.

Reply:
LTGM. Btw nit: shall we put it in `contrib` folder, or `frb_codegen/contrib`? Since if in the future we have another binary to distribute it will be a problem

Reply:
I don't really mind where they stay, so your call.

Reply:
Then maybe we can put it into `frb_codegen/contrib`